packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatAdapter.ts

```diff
@@ -369,12 +370,17 @@ export interface CallWithChatAdapterSubscriptions {
  *
  * @public
  */
-export interface CallWithChatAdapter
+export interface CallWithChatAdapterCommon
   extends CallWithChatAdapterManagement,
     AdapterState<CallWithChatAdapterState>,
     Disposable,
     CallWithChatAdapterSubscriptions {}
 
+/**
+ * @public
+ */
+export type CallWithChatAdapter = Omit<CallWithChatAdapterCommon, keyof ACSCallManagement> & ACSCallManagement;
```

Comment:
note: ARB is quite against the use of Omit and non-flat structures for public types, is there another way this could be defined?

Reply:
(we might need a seperate work item to define a flat structure for the adapter interface as it already extends a few things)